source/code/source/playfab/PlayFabAndroidHttpPlugin.cpp

```diff
@@ -435,15 +435,16 @@ namespace PlayFab
 
         // Call SetUrl
         {
-            const std::string requestUrl = GetUrl(requestTask);
+            CallRequestContainer* pfReqContainer = static_cast<CallRequestContainer*>(requestTask.requestContainer.get());
```

Comment:
It looks like pfReqContainer is only used to retrieve the GetFullUrl, i'd be a bit worried about any excess copies/casts if all we are doing is getting an internal member. Can we roll this call into the below line? 

caveat: it seems if the reqContainer doesn't get Initialize called (which I think we may be relying on a caller to do beforehand), theoretically .get() could return a nullptr which we may want to at least check for either a.) in the get or b.) right here (and any other get call)

Reply:
A few thoughts:
Right now, requestContainer is always defined, and must always be defined, because we use it below as well.
Here, I'm making a very strong assumption about how I can convert requestContainer from BaseCallRequestContainer to CallRequestContainer, which is an assumption made in every single HTTP layer at present.
Exaggerating this assumption here doesn't fundamentally change anything at present, because that assumption is everywhere, in every HTTP layer, and if that assumption were false, it'd break in dozens of other places too.
This is already a known "we'll have to fix this later" task that's already filed.

Reply:
requestContainer is a nullptr in the RequestTask object BUT 1.) it's only used in this plugin 2.) the only place where this object is used, we immediately call Initialize. Though note that this is in a try/catch block above that doesn't exit if it catches an exception. In some HIGHLY unlikely scenario where the construction worked but the Init didn't, the requestTask.requestContainer could still be a nullptr. 

What's that task number?